### ISSUE TITLE
did anyone finish sequence to sequence attention training? 

### ISSUE THREAD

**Issue by benlaitang** [andabi/deep-voice-conversion#82]
I write this referencing by https://github.com/keithito/tacotron, but it does not work.
the ground truth mel-spectrogram as input can work, but predicted mel failed.
Can anyone give me advises?

**Comment by wishvivek** [comment]
Yes, any clues on the Seq2Seq+Attention in this network will be great! Please update if anyone gets any solution. Thanks!
